.github/workflows/ci.yml

```diff
@@ -12,7 +12,33 @@ env:
   RUST_FEATURES: "rodio-backend,lyric-finder,media-control,image,notify,clipboard"
 
 jobs:
-  rust-ci:
+   msrv:
```

Comment:
can `msrv` be put as a step of `rust-ci` instead to avoid code duplication between those two?

Reply:
I'm seeing if the solutions [in this discussion](https://github.com/orgs/community/discussions/26284) to reuse matrices can be applied in a simple way here. Notable possible problem is that they don't use the same matrices as it is right now.
That's your main concern I assume, though the checkout and install packages steps could be abstracted away into another workflow, too.

Reply:
I thought you only need to move 
```
- name: Install cargo-hack
  uses: taiki-e/install-action@cargo-hack

- name: Run MSRV check 
  run: cargo hack check --rust-version --package spotify_player --all-targets --profile ci --features "${{ env.RUST_FEATURES }} ${{ matrix.OS_SPECIFIC_FEATURES }}"
```
to be in `rust-ci.steps`.

Reply:
yeah, one needs to check the action's logs if CI run fails anyway.